client/login/magic-login/style.scss

```diff
@@ -70,6 +70,10 @@
 	button {
 		width: 100%;
 	}
+
+	.form-button:last-child {
```

Comment:
It's generally safer to use classes rather than tags as selector, hence the new block.